isites_migration/templates/isites_migration/index.html

```diff
@@ -71,6 +109,27 @@
 {% block javascript %}
 <script>
     $(document).ready(function(){
+
+        // only show the more button if there are more than 3 rows
+        var rowCount = $('#select-site-table tr').length;
+        if(rowCount <= 3) {
+            console.log('boo');
+            $('#show-more').addClass('hidden');
+        }
+
+        // control how the more and less button are displayed
+        $('#show-more').click(function(){
```

Comment:
Seems like this entire bit of logic could be:
```
$('#show-more #show-less').click(function() {
  $('#show-more #show-less .collapsible').toggle();
});
```

...even better, `#show-*` could both have class `.toggle-collapsible`, and it's even shorter JS:
```
$('.toggle-collapsible').click(function() {
  $('.toggle-collapsible .collapsible').toggle();
});
```